documents/ogc-submission.adoc

```diff
@@ -0,0 +1,156 @@
+
+openEO is backed by various tools and resources that facilitate its use. These include a variety of server and client implementations, including a web interface for non-programmers and a server implementation that can run locally in a Python environment.
+
+== Relationship to other OGC Standards
+
+The openEO API builds on top of other standards and specifications.
+It's basis is HTTP with JSON as request and response encoding.
+The API specification reuses building blocks from OGC API - Common.
+It also implements the STAC API specification, which is based on the OGC API - Features standard.
+The "Secondary Web Services" in openEO allow to expose a wide-variety of visualization and download services using standards or specifications such as OGC WMS, OGC WMTS, OGC WFS, OGC WCS, OGC API - Tiles, OGC API - Maps, OGC API - Features, XYZ, and more.
+For Authentication the openEO API builds on top of the commonly used standards HTTP Basic and OpenID Connect.
+
+== Alignment with OGC Standards Baseline
+
+openEO continuously tries to align with OGC APIs (especially Common) and STAC. The openEO API standard fits well within the OGC API roadmap. 
+
+There are ongoing discussions to align OGC API - Processes and the openEO API better. Nevertheless, the openEO API primarily targets a user audience that OGC API - Processes doesn't cater for and thus they are not directly in competition to each other. 
+
+The openEO API is also considered as one building block for the emerging OGC Geodatacube API
+
+This specification enhances OGC operations and community involvement by providing a user-centric and less technical approach to big EO cloud processing, backed by an extensive set of pre-defined processes and a large ecosystem of open source servers and clients.
+
+== Evidence of implementation
+
+The following is a list of publically available implementations or deployments of the openEO API specification.
+
+Server deployments (9+):
+* openEO Platform - https://openeo.cloud
+* Copernicus Data Space Ecosystem - https://openeo.dataspace.copernicus.eu
+* EODC - https://openeo.eodc.eu
+* EURAC Research - https://openeo.eurac.edu
+* Google Earth Engine (maintained by a third-party) - https://earthengine.openeo.org
+* Mundialis - https://openeo.mundialis.de
+* Rasdaman - https://testbed19.rasdaman.com/rasdaman/openeo
+* Sentinel Hub - https://openeo.sentinel-hub.com/production
+* VITO - https://openeo.vito.be
+
+Client implementations (5+):
+* JavaScript - https://github.com/open-EO/openeo-js-client
+* Julia - https://github.com/open-EO/openeo-julia-client
+* Python - https://github.com/open-EO/openeo-python-client
+* R - https://github.com/open-EO/openeo-r-client
+* Browser-based User Interface - https://github.com/open-EO/openeo-web-editor
+
+More implementations can be found in the openEO GitHub organization: https://github.com/open-EO
```

Comment:
All these implementations are developed as open source software, and can be found, along with further implementations, in ...

Reply:
```suggestion
All these implementations are developed as open source software, and can be found, along with further implementations, in https://github.com/open-EO
```